test/powershell/Language/Classes/scripting.enums.tests.ps1

```diff
@@ -3,6 +3,11 @@
 #
 
 Describe 'enums' -Tags "CI" {
+    Context 'Enum definition' {
+        It "Enum values definition have space-insensitive syntax" {
+            { enum E0 {e=0} } | Should Not Throw

```

Comment:
Please follow the style of the other tests.  `Should Not Throw` is not useful for parser tests because the entire file is parsed before anything runs, so if it didn't parse correctly, no tests would run.

We should also ensure the initial value is correct when no space is used.